### PR TITLE
FAQ: add question go get TiDB fail.

### PR DESCRIPTION
**Pull request by tiancaiamao**
also update the Chinese document.
https://github.com/pingcap/docs-cn/pull/56

### CODE REVIEW

**Comment by ngaut** [comment]
PTAL @queenypingcap

**Comment by tiancaiamao** [comment]
updated @queenypingcap

**Comment by queenypingcap** [comment]
LGTM

**Comment by shenli** [comment]
@tiancaiamao Please resolve the conflict.

**Comment by shenli** [comment]
@tiancaiamao Any update?

**Comment by tiancaiamao** [comment]
PTAL @shenli

**Comment by tiancaiamao** [comment]
PTAL @shenli

File: FAQ.md [hunk 1]
```diff
@@ -156,6 +156,11 @@ Yes. Your applications can be migrated to TiDB without changing a single line of
 
 Yes. TiDB supports many popular storage engines, such as goleveldb and TiKV.
 
+## Does the `go get` or `go install` command work with TiDB?
+
+No. You can clone the [TiDB repository](https://github.com/pingcap/tidb) to your `$GOPATH/src/github.com/pingcap/tidb` and run the `make` command manually. TiDB is a project rather than a library. Its package management is very complex and the parser is generated using the [`parser.y`](https://github.com/pingcap/tidb/blob/master/parser/parser.y) file. So TiDB uses `Makefile` for package management.
```
Comment (shenli) [inline FAQ.md]: This is not true.
> TiDB uses `Makefile` for package management.

**Comment by queenypingcap** [comment]
@shenli PTAL.

**Comment by shenli** [comment]
LGTM

**Pull request merged**